pkg/cli/networkchanges_test.go

```diff
@@ -34,7 +34,7 @@ func generateNetworkChangeData(count int) {
 	now := time.Now()
 
 	for cfgIdx := range networkChanges {
-		networkID := fmt.Sprintf("a_new_network_change-%d", cfgIdx)
```

Comment:
I thought the existing name was fine for a **new** network change

Reply:
reverted